test/instances/storage.lua

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env tarantool
```

Comment:
Cite commit message
```lua
                 uri = {
                        'storage:storage@127.0.0.1:3301',
                        params = {
                            transport = 'plain',
                            ssl_ca_file = ca_file,
                        }
                    },
```

Personally I don't like mixture of lua array and lua table. Such table couldn't be serialized as json, yaml, msgpack properly.

Reply:
I took the format from EE tests. But you are right, better use `uri` key explicitly. Done.